backend/src/test/java/com/bakdata/conquery/integration/json/AbstractQueryEngineTest.java

```diff
@@ -89,8 +87,18 @@ protected void compareResults(Response csvResponse, SingleTableResult executionR
 						  .containsExactlyInAnyOrderElementsOf(expected);
 
 		// check that getLastResultCount returns the correct size
-		if (executionResult.streamResults().noneMatch(MultilineEntityResult.class::isInstance) && execution instanceof ManagedQuery managedQuery) {
-			assertThat(managedQuery.getLastResultCount()).as("Result count for %s is not as expected.", this).isEqualTo(expected.size() - 1);
+		if (executionResult.streamResults().noneMatch(MultilineEntityResult.class::isInstance)) {
+			long lastResultCount;
+			if (executionResult instanceof ManagedQuery managedQuery) {
```

Comment:
Hier wär die Frage ob das irgendwie in eine gemeinsame Abstraktion soll, allerdings wir die Methode so wie ich es gesehen hab nur von der `ManagedQuery` und `SqlManagedQuery` unterstützt.

Reply:
Das werdet ihr für die Formulare vermutlich eh nochmal neu anschauen müssen, wie man da den split macht. Deswegen kannst du das mit einem TODO erstmal so lassen.